src/e2e-test/features/pipeline.edit.feature

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright © 2022 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Integration_Tests
+Feature: Pipeline Edit
+
+  @PIPELINE_EDIT_TEST
+  @ignore
```

Comment:
Is there a way to turn on the feature flag in cucumber tests?

Reply:
I think removing the ignore tag when ff is removed will be fine